core/src/test/resources/configs/unified/9.0.xml

```diff
@@ -198,14 +198,16 @@
             <property name="foo">bar</property>
          </data-container>
       </local-cache>
-      <local-cache name="store-as-binary">
```

Comment:
This would should be converted to use `<memory> <binary/> <memory>` so it can pass the 70 test I mention above.